01-messaging.md

```diff
@@ -443,6 +449,60 @@ every message maximally).
 Finally, the usage of periodic `ping` messages serves to promote frequent key
 rotations as specified within [BOLT #8](08-transport.md).
 
+## Node Backup Storage
+
+Nodes that advertise the `peer_backup_storage` feature offer storing arbitrary
+data for their peers. The data stored must not exceed 32000 bytes, which lets
+it fit in existing lightning messages (e.g. a `commitment_signed` message with
+483 htlc signatures).
```

Comment:
Can you please double-check my calculations here? I *think* the biggest message that should be taken into account is `commitment_signed` with 483 HTLCs, with takes a bit more than 31000 bytes, so 32000 bytes is good (it leaves a bit more than 2000 bytes for future HTLC fields we'll add).